chains/solana/contracts/target/idl/ccip_router.json

```diff
@@ -1857,6 +1857,114 @@
         ]
       }
     },
+    {
+      "name": "LockOrBurnInV1",
```

Comment:
Note for reviewers: There is no real IDL changes due to this PR, it's just a reordering of some structs that I'm moving to different files so the auto-generated IDL lists them in different order